components/Document/DocumentActions.tsx

```diff
@@ -109,10 +110,11 @@ function DocumentActions({
             onSubmit={(flagReason, renderErrorMsg, renderSuccessMsg) => {
               flagGrmContent({
```

Comment:
If possible, let's do this overriding in the API level. I have done it there thus far. This way, it's all in the same layer and if we call a function from multiple places, we can ensure it overrides it at the source

Reply:
I like that. Thanks